src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -16,66 +24,260 @@
 	@Autowired
 	private EvalRepository dao;
 	
+	@Autowired
+	private EvalCommentRepository commentDao;
+
+    @Autowired
+    private QuestionLogic questionLogic;
+
+    @Autowired
+    private JsonValidation validation;
+	
+	@PersistenceContext
+    private EntityManager entityManager;
+	
 
 //CREATE----------------------------
+	@Override
+	public Eval createEval(Eval eval) {
+		
+		System.out.println(eval);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)